### PR TITLE
fix: import success message change

### PR DESCRIPTION
**Pull request by hujiahao-hjh**
## What this PR does / why we need it:
import success message change

## Does this PR introduce a user interface change?
- [x] Yes(screenshot is required)
- [ ] No
image

## Which versions should be patched?
release/1.1

## Which issue(s) this PR fixes:

Fixes # import success message change

### CODE REVIEW

File: shell/app/modules/project/pages/test-manage/case/header/import-file.tsx [hunk 1]
```diff
@@ -35,7 +35,11 @@ const ImportFile = ({ afterImport }: IProps) => {
 
   const onSuccess = ({ successCount }: any) => {
     if (isNumber(successCount)) {
-      message.success(i18n.t('project:imported {total} item successfully', { total: successCount }));
+      if (successCount === 0) {
+        message.success(i18n.t('project:importing'));
```
Comment (McDaddy) [inline shell/app/modules/project/pages/test-manage/case/header/import-file.tsx:39]: this onSuccess will call twice?

Comment (hujiahao-hjh) [reply shell/app/modules/project/pages/test-manage/case/header/import-file.tsx:39]: It won't. I tried it and it only show me the message once.

**Review by daskyrk — DISMISSED** [review]

File: shell/app/modules/project/pages/test-manage/case/header/import-file.tsx [hunk 2]
```diff
@@ -35,7 +35,7 @@ const ImportFile = ({ afterImport }: IProps) => {
 
   const onSuccess = ({ successCount }: any) => {
     if (isNumber(successCount)) {
-      message.success(i18n.t('project:imported {total} item successfully', { total: successCount }));
+      message.success(i18n.t('project:importing'));
```
Comment (daskyrk) [inline shell/app/modules/project/pages/test-manage/case/header/import-file.tsx:38]: will backend still return count? the message should be more meaningful like: start importing, please view detail in records.

Comment (hujiahao-hjh) [reply shell/app/modules/project/pages/test-manage/case/header/import-file.tsx:38]: The successCount returned by backend will only be 0.

**Review by daskyrk — APPROVED** [review]

**Pull request merged**